addons/website/static/src/components/configurator/configurator.js

```diff
@@ -9,7 +9,7 @@ import {svgToPNG} from 'website.utils';
 import {useService} from "@web/core/utils/hooks";
 import {renderToString} from "@web/core/utils/render";
 
-const { App, Component, onMounted, reactive, useEnv, useRef, useState, whenReady } = owl;
+const { App, Component, onMounted, onWillDestroy, reactive, useEnv, useRef, useState, whenReady } = owl;
```

Comment:
Commit message: task-XXX at the very end, after `[1]` commit reference (with blank lines around) please :wink: